packages/graph/hash_graph/lib/graph/src/knowledge/entity/mod.rs

```diff
@@ -25,12 +29,69 @@ impl fmt::Display for EntityId {
     }
 }
 
+/// An entity.
+///
+/// When expressed as JSON, this should validate against its respective entity type(s).
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, Component)]
 pub struct Entity {
     #[serde(flatten)]
     properties: HashMap<BaseUri, serde_json::Value>,
 }
 
+/// A record of an entity that has been persisted in the datastore, with its associated metadata.
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, Component)]
+pub struct PersistedEntity {
```

Comment:
Could we add 
`#[serde(rename_all = "camelCase")]` here? Just to make it consistent with the OpenAPI conventions

Reply:
Added, missed it during some refactoring. Thanks for spotting